### PR TITLE
[Command]schoolbook says "TV Receiver"

### PR DESCRIPTION
**Pull request by ftwbzhao**

### CODE REVIEW

File: Behavioral/Command/Tv/CommandChangeChannel.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    {
+        $this->output = $console;
+        $this->channelId = $channelId;
+    }
+
+    public function execute()
+    {
+        $this->output->changeChannel($this->channelId);
+    }
+}
```
Comment (malukenho) [inline Behavioral/Command/Tv/CommandChangeChannel.php]: Missing EOL EOF

File: Behavioral/Command/Tv/CommandOn.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    public function __construct(TvReceiver $console)
+    {
+        $this->output = $console;
+    }
+
+    public function execute()
+    {
+        $this->output->turnOn();
+    }
+}
```
Comment (malukenho) [inline Behavioral/Command/Tv/CommandOn.php]: Same above

**Comment by domnikl** [comment]
@ftwbzhao can you please fix the issues mentioned by @malukenho, so I can merge this PR?

**Comment by ftwbzhao** [comment]
[Command]schoolbook says "TV Receiver"

**Pull request closed without merging**